mm/alignment.py

```diff
@@ -789,3 +792,48 @@ def plot(self):
             plt.xticks(rotation=45)
 
             plt.savefig(plot_file)
+
+
+class GetBiggestAllophoneFileJob(Job):
+    """
+    Obtains the biggest allophone file from all allophone files passed as parameter.
+
+    All allophone files must be a common prefix of the biggest allophone file.
+    If this condition isn't met, the job will fail.
+    """
+
+    def __init__(self, allophone_files: List[tk.Path]):
+        self.allophone_files = allophone_files
+
+        self.out_biggest_allophone_file = self.output_path("allophone_file.txt")
+
+        self.rqmt = {"cpu": 1, "mem": 1.0, "time": 1.0}
+
+    def tasks(self):
+        yield Task("run", resume="run", rqmt=self.rqmt)
+
+    def run(self):
+        allophone_files = [util.uopen(allophone_file.get_path(), "rt") for allophone_file in self.allophone_files]
+        allophone_header_regex = re.compile(r"# Number of allophones: (\d+)")
```

Comment:
pipeline doesn't seem to like `regex`, maybe just use `line.split()[4]` instead?